app/gui2/src/util/ast/node.ts

```diff
@@ -1,5 +1,20 @@
 import type { NodeDataFromAst } from '@/stores/graph'
 import { Ast } from '@/util/ast'
+import { Prefixes } from '@/util/ast/prefixes'
+
+export let prefixes!: ReturnType<typeof makePrefixes>
+
+function makePrefixes() {
+  return Prefixes.FromLines({
+    enableRecording:
+      'Standard.Base.Runtime.with_enabled_context Standard.Base.Runtime.Context.Output __ <| __',
+  })
+}
+
+/** MUST be called after `initializeFFI`. */
+export function initializePrefixes() {
+  prefixes = makePrefixes()
+}
```

Comment:
If this must be called after `initializeFFI` we could encode that in the type system by making `initializeFFI` return a value that can't be constructed otherwise, and make that value an argument of this function.

But that might be overkill.

Reply:
yeah, i'm going to consider it as overkill for now. although it wouldn't be difficult to do it, it doesn't really match up with the rest of the codebase IMO.

Reply:
i mean, realistically in the actual app, both are called in the entrypoint in the correct order. so i don't think omitting it would be too big of a deal since there's only once place where they're called.